nacl-fast.js

```diff
@@ -2361,17 +2361,16 @@ nacl.setPRNG = function(fn) {
   // Initialize PRNG if environment provides CSPRNG.
   // If not, methods calling randombytes will throw.
   var crypto;
-  if (typeof window !== 'undefined') {
-    // Browser.
-    if (window.crypto && window.crypto.getRandomValues) {
-      crypto = window.crypto; // Standard
-    } else if (window.msCrypto && window.msCrypto.getRandomValues) {
-      crypto = window.msCrypto; // Internet Explorer 11+
-    }
-    if (crypto) {
+  if (typeof process === 'undefined' || process.browser) {
```

Comment:
The new logic will be:

```
 if (typeof process === 'undefined' || process.browser ||
      (typeof window !== 'undefined' && window.self === self)) {
```

Do you know if WebWorkers in Electron/nw have `process` defined?